joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/RubyIntermediateAst.scala

```diff
@@ -208,7 +208,7 @@ object RubyIntermediateAst {
   final case class StaticLiteral(typeFullName: String)(span: TextSpan) extends RubyNode(span) with LiteralExpr {
     def isSymbol: Boolean = text.startsWith(":")
 
-    def isString: Boolean = text.startsWith("\"")
+    def isString: Boolean = text.startsWith("\"") || text.startsWith("'")
```

Comment:
Perhaps something more robust
```scala
"^['\\\"]\\w*['\\\"]$".r.findFirstMatchIn(text).isDefined
```

Reply:
I agree this smells and is dodgy, but since the parsing of the static literal has already happened in ANTLR, there's no need to do it again. Unless there's a static literal in ruby other than single-quoted strings that begins with `'`.